firedrake/slate/slate.py

```diff
@@ -194,6 +201,17 @@ def rank(self):
     def coefficients(self):
         """Returns a tuple of coefficients associated with the tensor."""
 
+    @property
+    def coeff_map(self):
+        """A map from local coefficient numbers to the global
+        to the split global coefficient numbers.
+        The split coefficients are defined on the pieces of the originally mixed function spaces.
+        """
+        return tuple((n, tuple(range(len(c.split()))))
+                     if isinstance(c, Function) or isinstance(c, Constant)
```

Comment:
Under what circumstance are the coefficients not functions or constants?

Reply:
see comment on tsfc_interface